pypowsybl/network/impl/network.py

```diff
@@ -3893,6 +3893,82 @@ def create_2_windings_transformers(self, df: DataFrame = None, **kwargs: ArrayLi
         """
         return self._create_elements(ElementType.TWO_WINDINGS_TRANSFORMER, [df], **kwargs)
 
+    def create_3_windings_transformers(self, df: DataFrame = None, **kwargs: ArrayLike) -> None:
+        """
+        Creates three-winding transformers.
+
+        Args:
+            df: Attributes as a dataframe.
+            kwargs: Attributes as keyword arguments.
+
+        Notes:
+
+            Data may be provided as a dataframe or as keyword arguments.
+            In the latter case, all arguments must have the same length.
+
+            Valid attributes are:
+
+            - **id**: the identifier of the new transformer
+            - **rated_u0**: the rated voltage at the star bus
+            - **voltage_level1_id**: the voltage level where the new transformer will be connected on side 1.
+              The voltage level must already exist.
+            - **bus1_id**: the bus where the new transformer will be connected on side 1,
+              if the voltage level has a bus-breaker topology kind.
+            - **connectable_bus1_id**: the bus where the new transformer will be connectable on side 1,
```

Comment:
Can anybody explain to me what this field is for? I think it is not very clear, maybe we could improve a bit the documentation?

Reply:
I think that it is used if we want to create any network element that can be connected to a bus, by specifying which bus it can be connected to but not actually connecting it to this bus 
"bus1_id" defined -> the element is created connected to the bus 1
"connectable_bus1_id" defined -> the element is created not connected, when it is connected its bus is connectable_bus1_id

Reply:
Ok, thank you, I am going to try to explain that a bit better here